src/main/res/values/strings.xml

```diff
@@ -529,10 +529,10 @@
     <!-- option to show images in the gallery as square (instead of using correct width/height) -->
     <string name="square_grid">Square Grid</string>
     <string name="send_message">Send Message</string>
-    <!-- Placeholder %1$s will be replaced by the name of the contact changing their address. Placeholders %2$s and %3$s will be replaced by old/new email addresses. -->
+    <!-- Placeholder %1$s will be replaced by the name of the contact changing their address. Placeholders %2$s and %3$s will be replaced by old/new addresses. -->
     <string name="aeap_addr_changed">%1$s changed their address from %2$s to %3$s</string>
     <!-- the explanation is shown (1) as a modal dialog with the buttons "Cancel" and "Continue" as well as (2) as a device message -->
-    <string name="aeap_explanation">You changed your email address from %1$s to %2$s.\n\nIf you now send a message to a verified group, contacts there will automatically replace the old with your new address.\n\nIt\'s highly advised to set up your old email provider to forward all emails to your new email address. Otherwise you might miss messages of contacts who did not get your new address yet.</string>
+    <string name="aeap_explanation">You changed your address from %1$s to %2$s.\n\nIf you now send a message to a verified group, contacts there will automatically replace the old with your new address.\n\nIt\'s highly advised to set up your old address to forward all messages to your new address. Otherwise you might miss messages of contacts who did not get your new address yet.</string>
```

Comment:
I don't think it's necessary to change this string now (& require retranslations), since we will soon remove it: https://github.com/deltachat/deltachat-android/pull/3698

Reply:
retranslations won't be needed, translators will get a hint it should be re-reviewed (so they are aware of the change)

the thing i tuned also unrelated things more down is to _talk less_ also internally about email

(and it was easier when searching for email occurances while doing the PR :)

but yeah, i can re-add the original string, this is not primary UI anyways